clj/src/slipstream/ui/views/modal_dialogs.clj

```diff
@@ -141,6 +141,8 @@
                                           (u/enum-append-option :specify-for-each-node))
    :image-target-cloud            (some-> :available-clouds
                                           current-user/configuration)
+   :build-image-target-cloud      (some-> :available-clouds
+                                          current-user/configuration)
    :launch-mutable-run?           false
```

Comment:
Why do these expressions use `some->`?  These are equivalent (I think) to just `(:available-clouds current-user/configuration)`.